stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -213,8 +214,26 @@ function complete_symbol(sym::String, @nospecialize(ffunc), context_module::Modu
                 end
             end
         end
+
+        # Also, try abstract-interpreting propertynames
```

Comment:
Does this lead to duplicate suggestions for, e.g., concrete types?

Reply:
We should probably do this inference within `get_type` above.